Issue2180 boiler plant main controller

karthikeyad-pnnl opened the pull request.

JayHuLBL commented on Buildings/Examples/BoilerPlant/ClosedLoopTest.mo:
```diff
@@ -0,0 +1,281 @@
+    Diagram(coordinateSystem(preserveAspectRatio=false,extent={{-120,-100},{120,
+            100}})),
+    __Dymola_Commands(file=
+     "modelica://Buildings/Resources/Scripts/Dymola/Examples/Tutorial/Boiler/System6.mos"
```
There should be a .mos script for the example model `Buildings.Examples.BoilerPlant.ClosedLoopTest`.

JayHuLBL commented on Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Controller.mo:
```diff
@@ -0,0 +1,2251 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant;
+model Controller
```
This model is still not working. It failed to check in Dymola.

JayHuLBL commented:
@karthikeyad-pnnl I tried to run the model `Buildings.Examples.BoilerPlant.ClosedLoopTest` and `Buildings.Examples.BoilerPlant.ClosedLoopTest_legacy` with optimica but it failed. Then I realized that the the boiler plant main controller also failed to check in Dymola. 

Are you working on some refactoring and should I wait to check the two models?

karthikeyad-pnnl commented:
@JayHuLBL : Hi Jianjun, I am not sure why the main controller isn't working. I am adding an open loop validation file for it today. I will also create an .mos file for closed-loop validation model. I will tag you once it's up. Thanks for checking out the issue!

-Karthik

karthikeyad-pnnl commented:
@JayHuLBL : I have added an open-loop validation model for the main boiler plant controller. I have also added an mos file for the closed-loop test model.

karthikeyad-pnnl commented:
@milicag :
Hi Milica, this pull request is ready for review. It should contain all the boiler plant files. Please let me know if I should rebase it on a specific branch to remove any merge conflicts that may arise.

Thanks,
Karthik
